dockerfiles/oidc_e2e_setup_py310

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-stretch
+FROM python:3.10.4-bullseye
```

Comment:
stretch is no longer supported, so this updates the images to bullseye which is the current debian stable.